src/JSDomEnvironment.js

```diff
@@ -29,7 +31,8 @@ function JSDomEnvironment(config) {
   // use it (depending on the context -- such as TestRunner.js when operating as
   // a workerpool parent), this is the best way to ensure we only spend time
   // require()ing this when necessary.
-  this.global = require('./lib/jsdom-compat').jsdom().parentWindow;
+  this.global = require('./lib/jsdom-compat').jsdom().defaultView;
+  this.global.window = this.global;
```

Comment:
Yeah it's broken, `this` is undefined under the new vm setup. I don't have time to look into this right now but maybe someone can figure out how to set `this` to the correct value in the sandboxed context.